package.json

```diff
@@ -12,6 +12,7 @@
     "author": "CICA",
     "license": "MIT",
     "devDependencies": {
+        "ajv-formats-mobile-uk": "github:CriminalInjuriesCompensationAuthority/ajv-formats-mobile-uk#v0.0.1",
```

Comment:
should this be bumped to the latest version `1.0.0`?

Reply:
Done. Was just mirroring what's on q-template-application, didn't check the versions!